wp-content/themes/twentyseventeen-wp20/style.css

```diff
@@ -93,8 +93,8 @@ p {
 
 h1,
 h2,
+h2.wp20-event-group,
 h3,
-h3.wp20-event-group,
```

Comment:
I think we might need to keep this for both the `h2` and `h3`, since the map markers also use it:

![Screenshot 2023-04-12 at 10 20 48 AM](https://user-images.githubusercontent.com/484068/231535001-68c2c556-1b87-45d8-a161-992f8dcc1e9b.png)

![Screenshot 2023-04-12 at 10 20 09 AM](https://user-images.githubusercontent.com/484068/231534902-8daa5e2c-61af-430e-b798-83e822f6681e.png)

Or maybe the markers should also change to `h2`? They feel more like `h3` to me, though 🤔 

I wonder if instead we should add an `h2.screen-reader-text` above the map instead of changing the markers to `h2`? The same might apply to the list of events.

I don't have a strong opinion.

Reply:
Good point about those. I didn't have a popup open when I ran the scan. I'll try that and see what results.

Reply:
> I wonder if instead we should add an h2.screen-reader-text above the map instead of changing the markers to h2? The same might apply to the list of events.

Yeah I like this approach. Updated the PR, works well:

![Screen Shot 2023-04-13 at 8 56 45 AM](https://user-images.githubusercontent.com/1017872/231583315-18080c29-739c-4c21-88bc-a59635912dda.jpg)

Reply:
Looks great 👍🏻